build-scripts/cassandra-test-docker.sh

```diff
@@ -33,9 +40,23 @@ else
     DOCKER_IMAGE=$5
     TARGET=$6
     SPLIT_CHUNK=$7
+
+    # Setup JDK
+    java_version=$(java -version 2>&1 | awk -F '"' '/version/ {print $2}' | awk -F. '{print $1}')
+    if [ "$java_version" -ge 11 ]; then
+        java_version="11"
+        if ! grep -q CASSANDRA_USE_JDK11 build.xml ; then
+            echo "Skipping build. JDK11 not supported against $(grep 'property\s*name=\"base.version\"' build.xml |sed -ne 's/.*value=\"\([^"]*\)\".*/\1/p')"
+            exit 0
+        fi
+    else
+        java_version="8"
+    fi
+
     cat > env.list <<EOF
 REPO=$1
 BRANCH=$2
+JAVA_VERSION=${java_version}
```

Comment:
will `java_version` be always defined at this point?

Reply:
yes. it has to be `11` (line 47) or `8` (line 53).

Reply:
yeah, indentation for `cat` fooled me.. :facepalm: